sources/Inputs/include/Behaviours.h

```diff
@@ -72,5 +73,43 @@ struct Generator {
   double pmax;                             ///< maximum active power
 };
 
+class HvdcLine;
+/**
+ * @brief Converter interface behaviour
+ */
+struct ConverterInterface {
+  using ConverterInterfaceId = std::string;  ///< alias for id
+  using BusId = std::string;                 ///< alias for bus id
+
+  /**
+   * @brief Constructor
+   *
+   * @param converterId the id of the converter
+   * @param busId the id of the bus
+   * @param voltageRegulationOn optional boolean for the voltage regulation parameter, only used for VSC converters
+   * @param hvdcLine the hvdc line this converter is contained into
+   */
+  explicit ConverterInterface(const ConverterInterfaceId& converterId, BusId busId, boost::optional<bool> voltageRegulationOn = {},
+                              boost::shared_ptr<dfl::inputs::HvdcLine> hvdcLine = nullptr) :
+      converterId{converterId},
+      busId{busId},
+      voltageRegulationOn{voltageRegulationOn},
+      hvdcLine{hvdcLine} {}
+  /**
+   * @brief Determines if two converter interfaces are equal
+   *
+   * @param other the other converter to be compared against
+   * 
+   * @return status of the comparaison
+   */
+  bool operator==(const ConverterInterface& other) const {
```

Comment:
Are you sure of this syntax for the equality operator ? I only know the version with the protoype:
bool operator==(const &, const &)

Reply:
I verified in debug that the function was indeed called with a syntax like ConverterInterface == ConverterInterface2

Reply:
OK then